planetary-social/nos
# Flag user strings

## Issues covered
Part 1 of #1493 

## Description
This adds localised strings and arrays for the new flag account flow. It also adds an existing card shadow modifier to the cards in the new .

This started out as just a string localisation PR, but it turns out that the flag content and flag account have similar flag categories with just an extra one for the flag account categories. That is why, this contains more changes to refactor the `FlagOption` model and other views that make use of it.

## How to test
Please check that the localised strings are consistent with [figma](https://www.figma.com/design/s0qf4VmyQygydP8MIQazZc/Nos?node-id=8712-20089&t=m4lMymxYI57OYSEh-4).
Please also make sure that the new content flag flow still works.
